docs/portal.properties-Reference.md

```diff
@@ -61,52 +75,59 @@ authorization=true
+These data formats are described in the [cBioPortal MAF specifications](File-Formats.md#extending-the-maf-format). 
+
+#### Enabling custom annotations in the OncoPrint
+To enable functionality for one or both types of custom annotations, enter values for the following properties. These labels will appear in the OncoPrint's "Mutation color" menu.
 ```
- oncoprint.custom_driver_annotation.default=false
+oncoprint.custom_driver_annotation.binary.menu_label=Custom driver annotation
+oncoprint.custom_driver_annotation.tiers.menu_label=Custom driver tiers
 ```
 
-By default, when at least one of the custom annotations is enabled in portal.properties and the required columns are present in MAF files, custom annotations are automatically selected. You can change this behaviour and set the following properties to "false" to automatically unselect one or both custom driver/passenger mutations (by default, those properties are set to true):
+#### Automatic selection of custom annotations
+By default, when at least one of these custom annotations is enabled in `portal.properties` and the required columns are present in MAF files, custom annotations are automatically selected instead of OncoKB and Hotspots. To disable this default behavior and have OncoKB and Hotspots as the default selected annotations, set the following properties to `false`.
 ```
-oncoprint.custom_driver_annotation.default=false
-oncoprint.custom_driver_tiers_annotation.default=false
+oncoprint.custom_driver_annotation.default=true
+oncoprint.custom_driver_tiers_annotation.default=true
 ```
 
-Independently of the custom annotations, OncoKB and Hotspots are always automatically selected as annotation source. If you want to disable them, set the following property to false:
-```
-oncoprint.oncokb_hotspots.default=false
-``` 
-However, you can also set the same property to "custom". This will automatically select OncoKB and Hotspots only if there are no custom annotation driver and passenger mutations:
+#### Automatic selection of OncoKB annotations
+OncoKB and Hotspots are by default automatically selected as annotation source. To disable this, set the following property to `false`. To only select OncoKB and Hotspots when there are no custom driver mutation annotations, set this property to `custom`.
 ```
 oncoprint.oncokb_hotspots.default=custom
-```
+``` 
 
-By default, the putative passenger mutations are not hidden. If you want to automatically hide Putative Passenger Mutations, set this property to true:
+#### Automatic hiding of putative passenger mutations
+By default, the selection box to hide putative passenger mutations is unchecked. If you want to automatically hide Putative Passenger Mutations, set this property to `true`:
 ```
-oncoprint.hide_passenger.default=true
+oncoprint.hide_passenger.default=false
```

Comment:
@sandertan same here. Maybe the idea is that you're thinking it is false by default and I should set it to true if I read the text above, but that seems confusing to me.

Reply:
By default Putative Passenger Mutations are **not** hidden in the portal. So to represent this behaviour with the new property, this property should be `false`. 

We can also keep all these properties commented out (as it was intially, and I have in latest version of this branch), to prevent them from having any influence on default installations.